tests/python_functional/functional_tests/logpath/test_multiple_embedded_logpaths.py

```diff
@@ -96,4 +96,4 @@ def test_multiple_embedded_logpaths(config, syslog_ng, bsd_formatter):
 
     # no messages should arrived into destination4,
     # no source() or flags(catch-all) is added
-    assert file_destination4.get_path().exists() is False
+    assert file_destination4.get_connection().exists() is False
```

Comment:
* I do not favor specific names here, because the concept of this PR to give a "general" API to reach "resource options" for each drivers.

* In my opinion this is the same case as we did not use the `read_file()` and `write_file()` methods, instead of it we use a "general" names: `read_log()` and `write_log()`.

* If the name `get_connection()` is not trivial here we can change it to another: `get_resource()`.

* One more option is here to use both names (as an alias) : `get_connection()`/`get_resource()` and `get_path()`